mantidproject/mantid
# Update python path in the pycharm environment file

**Description of work.**
This updates the python path in the .env for pycharm on windows.

**To test:**
Make sure you are using a python 3 build of mantid and you have pycharm set up so it uses the .env file (instruction [here](http://developer.mantidproject.org/GettingStartedWithPyCharm.html))
Run any test - it should run without giving an error
Run workbench from pycharm - should run without error.

*There is no associated issue.*

*This does not require release notes* because **it is only a change for developers**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
